docs/CONTRIBUTING.md

```diff
@@ -102,6 +102,49 @@ This usually involved editing the `Containerfile`. Most techniques for building
 
 Check out CoreOS's [layering examples](https://github.com/coreos/layering-examples) for more information on customizing.
 
+## Using Blue-build locally
+### Requirements
+
+- [Blue-build CLI](https://github.com/blue-build/cli)
+- Podman
+- git
+
+Secureblue already includes `bluebuild` but running locally requires customizing policy.json for your user to allow pulling a few unsigned images.
+
+#### Policy.json configuration
+
+On secureblue only pre-configured signed images are allowed to be pulled. Four repos need to be configured:
+
+- docker.io/mikefarah/yq `Unsigned`
+- gcr.io/projectsigstore/cosign `Signed`
+- ghcr.io/blue-build/cli `Unsigned`
```

Comment:
same for bluebuild, I believe they are signed too, so we can add their key and repos

Reply:
Their repo has a cosign.pub but `cosign tree` couldn't find any signatures. So I left them as unsigned.

Reply:
oh damn. it'd be worth creating an issue on bluebuild then, i think they're intended to be signed